databricks/koalas/frame.py

```diff
@@ -6448,14 +6448,12 @@ def swaplevel(self, i=-2, j=-1, axis=0) -> "DataFrame":
 
         return DataFrame(internal)
 
-    def swapaxes(
-        self, i: Union[str, int] = 0, j: Union[str, int] = 1, copy: bool = True
-    ) -> "DataFrame":
+    def swapaxes(self, i: Union[str, int], j: Union[str, int], copy: bool = True) -> "DataFrame":
         """
         Interchange axes and swap values axes appropriately.
 
-        .. note:: This method is based on an expensive operation due to the nature
-            of big data. Internally it needs to generate each row for each value, and
+        .. note:: This method, if applied to a DataFrame, is based on an expensive operation due to
```

Comment:
We don't need `if applied to a DataFrame`?

Reply:
Sounds good, removed~